tools/releaseBuild/setReleaseTag.ps1

```diff
@@ -36,6 +36,7 @@ function New-BuildInfoJson {
     ## Get the UTC time and round up to the second.
     $dateTime = [datetime]::UtcNow
     $dateTime = [datetime]::new($dateTime.Ticks - ($dateTime.Ticks % [timespan]::TicksPerSecond), $dateTime.Kind)
+    $dateTime = $dateTime.ToString("o")
```

Comment:
The purpose is to get the precision up to second, like `2019-12-03T22:04:15Z`. `ToString("o")` gives you `2019-12-03T22:04:15.0000000Z`.

We don't need to address this now, as the preview.json, stable.json and lts.json will have the `ReleaseDate` re-set at the end of the release pipeline, where `pwsh` is used.
I think we need to make the build agent pre-install pwsh eventually.